knowledge-base/unable-to-get-report-parameters-report-cannot-be-resolved.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Unable to get report parameters - Report reportName cannot be resolved
+description: Unable to get report parameters - Report reportName cannot be resolved
+type: troubleshooting
+page_title: Unable to get report parameters - Report reportName cannot be resolved
+slug: unable-to-get-report-parameters-report-cannot-be-resolved
+position: 
+tags: Azure, Linux
+ticketid: 1532722
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+
+Trying to display a report in a report viewer through the use of a [Reporting web service](../telerik-reporting-rest-conception) may result in the following error being displayed inside the report viewer.
+
+## Error Message
+
+```
+Unable to get report parameters.
+Report reportName.trdp/x cannot be resolved.
+```
+
+## Cause\Possible Cause(s)
+
+- The report definition file cannot be resolved by the reporting service because it cannot be found at the given path. 
+
+- The service does not have permissions for the folder used to store the report definitions and thus the reports cannot be resolved.
+
+- If the application with the reporting web service is deployed in an environment that heavily restricts the GDI APIs. An example of such a case would an Azure environment with **lower** than **Basic** [App Service plan](https://azure.microsoft.com/en-us/pricing/details/app-service/windows/).
+
+- If the application with the reporting web service is running on a Linux system. This is because the GDI APIs are not available **at all** on Linux and substitite libraries must be installed in order for the reports to be rendered.  
+
+## Solution
+
+- Make sure that the path used by the ReportSourceResolvers is correct in every environment under which the reporting service is run and that the folder with the reports is accessible.
+
+- The problem may be related to folder permissions to access the folder. For example, make sure that the "**Copy to output directory**" setting of the report file is set to **True**. This setting may be configured by selecting the file inside Visual Studio. It should be inside the **Properties** window of that file.
```

Comment:
I think "The problem may be related to folder permissions" is enough, " to access the folder" is somehow repeating the previous one. Or you may change it to "The problem may be related to folder access permissions"